languages.toml

```diff
@@ -1964,3 +1964,17 @@ roots = []
 [[grammar]]
 name = "ini"
 source = { git = "https://github.com/justinmk/tree-sitter-ini", rev = "4d247fb876b4ae6b347687de4a179511bf67fcbc" }
+
+[[language]]
+name = "haxe"
+scope = "source.haxe"
+injection-regex = "haxe"
+file-types = ["hx"]
+comment-token = "//"
+indent = { tab-width = 2, unit = "\t" }
+roots = ["hxml", "khafile.js"]
```

Comment:
I don't think `roots` supports file extensions. I'm not very familiar with Haxe but could we use the `haxelib.json` file for this?

Reply:
I included the `hxml` root because I noticed `sln` and `csproj` in the roots for C#, but that also may be an error in the first place if indeed `roots` doesn't work like that.

Reply:
It looks like we don't support extensions for roots; it's a `Path::exists` call: https://github.com/helix-editor/helix/blob/4ec2a21c6e21ab4e515f1bd7ee1049094af2a6b2/helix-core/src/lib.rs#L68